tools/ci_build/build.py

```diff
@@ -1547,6 +1547,9 @@ def run_onnxruntime_tests(args, source_dir, ctest_path, build_dir, configs):
                 build_dir, config, "external", "tvm", config))
         if args.use_tensorrt:
             dll_path_list.append(os.path.join(args.tensorrt_home, 'lib'))
+        if args.build_eager_mode and is_windows():
+            import torch
+            dll_path_list.append(os.path.join(os.path.dirname(torch.__file__), 'lib'))
```

Comment:
Would you please explain a little bit why this is needed?

Reply:
Why do you need to load a DLL from pytorch dir? Why pytorch can't help you load it?  Do you know this totally doesn't work for new python version >=3.8?

Reply:
In eager mode, while importing onnxruntime it is unable to find torch libraries. By adding the path of torch lib in the PATH variable resolve this issue.

Reply:
@snnn , in short, with eager mode, onnxruntime pybind module will take dependency to torch shared library. so user need to import torch first then import onnxruntime. but we don't want to update all of our python test files. The workaround here is to add the torch.so path to PATH so python can find it. We are aware it doesn't work for python >= 3.8, do you have any suggestion?

Reply:
Good. Let's put some comments there.